files/default/setup-ephemeral-drives.sh

```diff
@@ -13,124 +11,189 @@ set -x
 
 . /etc/parallelcluster/cfnconfig
 
-# Error exit function
-function error_exit () {
-  script=`basename $0`
-  echo "parallelcluster: $script - $1"
-  logger -t parallelcluster "$script - $1"
+LVM_VG_NAME="vg.01"
+LVM_NAME="lv_ephemeral"
+LVM_PATH="/dev/${LVM_VG_NAME}/${LVM_NAME}"
+LVM_ACTIVE_STATE="a"
+FS_TYPE="ext4"
+MOUNT_OPTIONS="noatime,nodiratime"
+INPUT_MOUNTPOINT="${ephemeral_dir}"
```

Comment:
minor: I'd add a comment to explain that ephemeral_dir variable is coming from cfnconfig file

Reply:
done